client/src/components/campaign/round-new/round-new.html

```diff
@@ -0,0 +1,165 @@
+            <md-input-container class="no-margin">
+              <textarea aria-label="Round directions"
+                ng-model="$ctrl.round.directions"
+                rows="2" ></textarea>
+            </md-input-container>
+          </div>
+          <div flex></div>
+          <div class="round__settings" flex="30">
+            <h4>Round File Settings</h4>
+            <p layout="column" layout-align="start stretch">
+              <span layout="row" layout-align="space-between start"
+                  ng-repeat="(name, value) in $ctrl.round.config">
+                <span ng-if="name !== 'min_resolution'">{{ ::name | translate }}</span>
+                <span ng-if="value === false || value === true">
+                  <md-checkbox
+                    ng-model="$ctrl.round.config[name]"
+                    aria-label="{{ ::name }}">
+                </span>
+                <md-input-container ng-if="name === 'min_resolution'">
+                  <label>Minimal resolution</label>
+                  <input type="number" ng-model="$ctrl.round.config.min_resolution">
+                </md-input-container>
+              </span>
+            </p>
+          </div>
+        </div>
+        
+        <div class="round__jurors"
+            layout="row" layout-align="center stretch">
+          <div flex>
+            <h4>Quroum</h4>
+            <p class="muted">The number of jurors that must vote on each image</p>
+            <md-input-container class="no-margin-bottom">
+              <label>Quorum</label>
+              <input type="number"
+                  ng-model="$ctrl.round.quorum"
+                  ng-required="true">
+            </md-input-container>
+            <h4>Jurors</h4>
+            <mont-user-list
+              ng-model="$ctrl.round.jurors">
+            </mont-user-list>
+          </div>
+          <div flex="30"
+              ng-if="$ctrl.index > 1">
+            <h4>Threshold</h4>
+            <p class="muted">Minimal average rating for photo</p>
+            <md-select ng-model="$ctrl.round.threshold">
+              <md-option ng-required="true"
+                  ng-repeat="(key, value) in $ctrl.prevRound.details.thresholds"
+                  ng-value="key">
+                 {{ (key * 10).toFixed(2) }} / 10 ({{ value }} images total)
+              </md-option>
+            </md-select>
+          </div>
+        </div>
+      </fieldset></md-card-content>
+    </md-card>
+    <div class="round__actions"
+          layout="row" layout-align="end center">
+      <md-button
+          ng-disabled="$ctrl.loading"
+          ng-click="$ctrl.cancelAddRound()">
+        <md-icon>clear</md-icon>
+        Cancel
+      </md-button>
+      <md-button class="md-primary"
+          ng-click="$ctrl.prevRound ? $ctrl.advanceRound() : $ctrl.addRound()"
+          ng-disabled="$ctrl.roundEditForm.$invalid || $ctrl.loading || !$ctrl.round.jurors.length">
+        <md-icon>check</md-icon>
+        Add Round
```

Comment:
This definitely needs a loading spinner.  It can take a while for a big category.